UPGRADE.md

```diff
@@ -2,6 +2,18 @@
 
 ## dev-master
 
+### Add key property to Role entity
+
+To allow for referencing `Role` entities by a human readable string, a `key` property was added to the `Role` entity.
+Furthermore, if the `key` is set, it is used for generating the identifier of a role instead of the `name` property.
+The following statements update the database to include the new property:
+
+```sql
+DROP INDEX UNIQ_13B749A05E237E06 ON se_roles;
```

Comment:
we should keep the uniqueness of the name

Reply:
not sure about that 🤔 why would you keep it?

Reply:
Role Name think also from UX be unique.

Reply:
Also we still need to use the name and so should not change the behaviour is somebody rely on the RoleNameAlreadyExistsExcpetion

Reply:
well i will trust you here 🙂

Reply:
I hope I don't disappoint you.

Reply:
me too 😉